agent/agent-gc-monitor/gc-monitor-tests/src/test/java/com/microsoft/gcmonitortests/VariousCollectorsTest.java

```diff
@@ -6,6 +6,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.condition.JRE.JAVA_11;
 import static org.junit.jupiter.api.condition.JRE.JAVA_13;
+import static org.junit.jupiter.api.condition.JRE.JAVA_17;
```

Comment:
does it need java 21 and java 23 test here since you added java 17?

Reply:
this import is for an exclusion, see below:

> @EnabledForJreRange(max = JAVA_17)